runners/flink/src/main/java/org/apache/beam/runners/flink/LyftFlinkStreamingPortableTranslations.java

```diff
@@ -325,21 +324,32 @@ private static long parseDateTime(String datetime) {
       Iterator<JsonNode> iter = events.elements();
       long timestamp = Long.MAX_VALUE;
       while (iter.hasNext()) {
-        JsonNode occurredAt = iter.next().path(EventField.EventOccurredAt.fieldName());
+        JsonNode event = iter.next();
```

Comment:
This logic is getting pretty complicated; would you mind adding a small comment summarizing the strategy?

Reply:
done